src/DependencyInjection/Configuration.php

```diff
@@ -111,6 +111,7 @@ private function addPersistenceSection(NodeBuilder $treeBuilder)
                         ->canBeEnabled()
                         ->children()
                             ->scalarNode('manager_name')->defaultNull()->end()
+                            ->scalarNode('content_basepath')->defaultValue('/cms/content')->end()
```

Comment:
No, we can leave the default here as is. If CoreBundle is registered, the default will be overriden by the value that is prepended. If CoreBundle isn't available, the default value will be used.